### DISCUSSION TITLE
The First Release!

### DISCUSSION THREAD

**Discussion by its-darsh** [Fabric-Development/fabric#48] [Announcements]
## What's Changed
* Update PKGBUILD to reflect new changes by @killertofus in https://github.com/Fabric-Development/fabric/pull/1
* fixed examples/side_panel by @Deeepkk1 in https://github.com/Fabric-Development/fabric/pull/2
* feat: add scale widget by @muhchaudhary in https://github.com/Fabric-Development/fabric/pull/11
* feat: new utility for getting desktop applications by @its-darsh in https://github.com/Fabric-Development/fabric/pull/13
* feat: add bluetooth service and provide example by @muhchaudhary in https://github.com/Fabric-Development/fabric/pull/14
* feat: add argument `keyboard_mode` to `wayland.Window` by @muhchaudhary in https://github.com/Fabric-Development/fabric/pull/16
* fix: match with new hyprland changes by @its-darsh in https://github.com/Fabric-Development/fabric/pull/20
* Fix lambdas in vertical bar example by @levnikmyskin in https://github.com/Fabric-Development/fabric/pull/23
* fixed vertical bar example button matching logic by @h4m6urg1r in https://github.com/Fabric-Development/fabric/pull/24
* fix: Make Hyprland module compatible with +0.40.0 by @Axenide in https://github.com/Fabric-Development/fabric/pull/26
* feat: add markup support for labels  by @Axenide in https://github.com/Fabric-Development/fabric/pull/27
* Update PKGBUILD by @mihokei in https://github.com/Fabric-Development/fabric/pull/30
* Add "include_hidden" parameter to get_desktop_applications function by @muhchaudhary in https://github.com/Fabric-Development/fabric/pull/32
* Fix: Typo in Label property by @Axenide in https://github.com/Fabric-Development/fabric/pull/33
* feat: add 'invert_scroll' and 'allow_empty' to Workspaces by @Axenide in https://github.com/Fabric-Development/fabric/pull/34
* feat: Update HyprlandWorkspaces on focusedmon event by @autumnWindigo in https://github.com/Fabric-Development/fabric/pull/41

## New Contributors
* @killertofus made their first contribution in https://github.com/Fabric-Development/fabric/pull/1
* @Deeepkk1 made their first contribution in https://github.com/Fabric-Development/fabric/pull/2
* @muhchaudhary made their first contribution in https://github.com/Fabric-Development/fabric/pull/11
* @levnikmyskin made their first contribution in https://github.com/Fabric-Development/fabric/pull/23
* @h4m6urg1r made their first contribution in https://github.com/Fabric-Development/fabric/pull/24
* @Axenide made their first contribution in https://github.com/Fabric-Development/fabric/pull/26
* @mihokei made their first contribution in https://github.com/Fabric-Development/fabric/pull/30
* @autumnWindigo made their first contribution in https://github.com/Fabric-Development/fabric/pull/41

**Full Changelog**: https://github.com/Fabric-Development/fabric/commits/v0.0.1

## Community Messages (Discord)
"Sofa Approved" - Sofa
"It works, despite GNOME's best efforts" - Ahmad (@muhchaudhary)

---
## Enough Yapping. Here's The Release Cat!

![Cat Image by @thatstepan (discord)](https://github.com/user-attachments/assets/6c293174-3cbf-4c4b-bfc8-42a83e53ef36)

<hr /><em>This discussion was created from the release <a href='https://github.com/Fabric-Development/fabric/releases/tag/v0.0.1'>The First Release!</a>.</em>

**Comment by Axenide** [comment]
# HYPE
